src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv.sarif

```diff
@@ -0,0 +1,1353 @@
+                  "endColumn": 41
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Exceptions should not be dropped.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/monitor/FileAlterationMonitor.java",
+                "region": {
+                  "startLine": 189,
+                  "startColumn": 15,
+                  "endLine": 189,
+                  "endColumn": 56
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "error",
+          "message": "This BOMInputStream is not always closed on method exit.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/input/XmlStreamReader.java",
+                "region": {
+                  "startLine": 269,
+                  "startColumn": 36,
+                  "endLine": 269,
+                  "endColumn": 108
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "error",
+          "message": "This BOMInputStream is not always closed on method exit.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/input/XmlStreamReader.java",
+                "region": {
+                  "startLine": 270,
+                  "startColumn": 36,
+                  "endLine": 270,
+                  "endColumn": 81
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "note",
+          "message": "Method doCopyFile ignores exceptional return value of File.setLastModified.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/FileUtils.java",
+                "region": {
+                  "startLine": 1145,
+                  "startColumn": 13,
+                  "endLine": 1145,
+                  "endColumn": 60
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "note",
+          "message": "Method doCopyDirectory ignores exceptional return value of File.setLastModified.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/FileUtils.java",
+                "region": {
+                  "startLine": 1430,
+                  "startColumn": 13,
+                  "endLine": 1430,
+                  "endColumn": 58
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "This field is in a serializable class,  but is not serializable itself because CharSequence is not serializable.",
```

Comment:
I assume you're asking because we'd offer to fix it if they were. I'll ask them.

---
In reply to: [90076792](https://github.com/Microsoft/sarif-sdk/pull/545#discussion_r90076792) [](ancestors = 90076792)